.github/workflows/coder.yaml

```diff
@@ -371,6 +371,11 @@ jobs:
             --package-linux \
             linux:amd64
 
+          # build windows amd64 packages
+          ./scripts/build_go_matrix.sh \
+            --output ./dist/ \
+            windows:amd64
```

Comment:
You can append this to the above execution and it will run in parallel. This should speed up the deploy.

Reply:
Ok.  I just blindly assumed that the `--package-linux` did something undesirable for Windows, but I didn't actually look.